massgov/mayflower-patternlab
# DP-8648 fix gap in image background on large widths

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Fix the vertical gap on the image background for wide widths on the Elected Official Organization page when the headline is 3 rows.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8648)


## Steps to Test


1. View `http://localhost:3000/?p=pages-organization-elected-official` 
2. Edit the html to make the headline span 3 rows
3. Ensure that the image background maintains coverage of the background (is flush with the colored, angled background)

## Screenshots
![image-gap-bugfix](https://user-images.githubusercontent.com/1397914/39006610-97f0a776-43d1-11e8-8ef5-40b927ff0014.gif)



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
